`rocm_sdk.tests.core_test.ROCmCoreTest.testConsoleScripts` fails in `Release portable Linux PyTorch Wheels` workflow

All release workflows for which test machines are available (e.g. https://github.com/ROCm/TheRock/actions/runs/19029964424/job/54343771081) fail with:
```
FAIL: testConsoleScripts (rocm_sdk.tests.core_test.ROCmCoreTest.testConsoleScripts) [Check console-script amdflang]
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/__w/TheRock/TheRock/.venv/lib/python3.11/site-packages/rocm_sdk/tests/core_test.py", line 126, in testConsoleScripts
    self.fail(
AssertionError: Expected 'clang LLVM compiler' in console-script amdflang outuput:
OVERVIEW: flang LLVM compiler

USAGE: flang-22 [options] file...
```

--- a/build_tools/packaging/python/templates/rocm/src/rocm_sdk/tests/core_test.py
+++ b/build_tools/packaging/python/templates/rocm/src/rocm_sdk/tests/core_test.py
@@ -33,7 +33,7 @@ COMMON_CONSOLE_SCRIPT_TESTS = [
     ("amdclang++", ["--help"], "clang LLVM compiler", True),
     ("amdclang-cpp", ["--help"], "clang LLVM compiler", True),
     ("amdclang-cl", ["-help"], "clang LLVM compiler", True),
-    ("amdflang", ["--help"], "flang LLVM compiler", True),
+    ("amdflang", ["--help"], "LLVM compiler", True),
     ("amdlld", ["-flavor", "ld.lld", "--help"], "USAGE:", True),
     ("hipcc", ["--help"], "clang LLVM compiler", True),
     ("hipconfig", [], "HIP version:", True),

